modules/color-picker/main/index.coffee

```diff
@@ -227,7 +227,7 @@ class ColorPicker extends hx.EventEmitter
           if endCallback isnt null then endCallback(elem)
           dragging = false
 
-        dispose: ->
+        dispose = ->
```

Comment:
Should be ok I think. The only place `disposed` is assigned to is in the `dispose` function which isn't used, so `disposed` is always going to be null.